.bitmap

```diff
@@ -1020,6 +1020,12 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/design/ui/alert-card"
     },
+    "ui/apply-providers": {
```

Comment:
I think we should group these new components under a `ui/preview` or `ui/docs` namespace